Umbraco-Cloud/Upgrades/Manual-Upgrades/Manual-CMS-upgrade.md

```diff
@@ -14,117 +14,29 @@ Make sure you can run your Cloud project locally and restore content and media.
 
 ## Get the latest version of Umbraco
 
-* [Download the latest version Umbraco CMS from Our](https://our.umbraco.com/download/)
-* Unzip the folder to your computer
-* Copy the following folders from the unzipped folder to your Cloud project folder:
-    * `/bin`
-    * `/Umbraco`
+To get the latest version of Umbraco you will need to upgrade the site using NuGet.
```

Comment:
As this article is centered around Umbraco Cloud I think it would be a good idea to start with cloning the project or pulling to get latest changes BEFORE installing any nuget packages.
And furthermore, as there is a standard folder format it would be a good idea to include something like "navigate to the the /src/UmbracoProject/ folder, which has the csproj with the CMS nuget packages installed"